service-front/module/Application/src/Model/Service/Session/SessionManagerSupport.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Application\Model\Service\Session;
+
+use Laminas\Session\Container;
+use Laminas\Session\SessionManager;
+
+class SessionManagerSupport
+{
+    public function __construct(private SessionManager $sessionManager)
+    {
+    }
```

Comment:
Is this here to satisfy an interface somewhere this is used? Asking as we're not doing anything with the `SessionManager` that's passed in.

Reply:
I think constructor promotion was added in PHP 8.0, so maybe post you moving to golang?  This is an implicit setting of the private var SessionManager to reduce boilerplate. https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion